### PR TITLE
feat(SDB): create how tos ServerlessDB MTA-3455

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [x] Check that the commit messages match our requested structure.
- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description
Please describe your pull request.

### CODE REVIEW

File: serverless/sql-databases/how-to/add-users.mdx [hunk 1]
```diff
@@ -20,4 +20,44 @@ Users can connect to a database and access its data. Each one has a different se
...
+
+6. Click **Validate**.
+
+## How to add an existing application to a Serverless SQL Database
+
+1. Click **Identity and Access Management (IAM)** from the top-right of your [**Organization Dashboard**](https://console.scaleway.com/organization) in the Scaleway console. The **Users** tab of the [Identity and Access Management dashboard](https://console.scaleway.com/iam/users) displays.
+
+2. Open the **Applications** tab.
+
+2. Click the **name** of the application you want to add to the database.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:51]: double step 2 here (and following steps misaligned)

**Review by bene2k1 — APPROVED** [review]

File: serverless/sql-databases/how-to/add-users.mdx (hunk 1 above)
Comment (bene2k1) [reply serverless/sql-databases/how-to/add-users.mdx:51]:
```suggestion
3. Click the **name** of the application you want to add to the database.
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+3. Click the **Groups & Policies** tab.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:53]:
```suggestion
4. Click the **Groups & Policies** tab.
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+4. Click **+ Attach a policy**. The policy attachment wizard displays.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:55]:
```suggestion
5. Click **+ Attach a policy**. The policy attachment wizard displays.
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+
+5. Choose **Select an unassigned policy**, then, select the previously created policy from the drop-down menu.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:57]:
```suggestion
6. Choose **Select an unassigned policy**, then, select the previously created policy from the drop-down menu.
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 5]
```diff
@@ -0,0 +1,63 @@
...
+
+    <Message type="note">
+        You can also [create a new policy](serverless/sql-databases/how-to/manage-permissions/) or duplicate an existing one to manage the permissions of an application.
+    </Message>
+
+6. Click **Validate**.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:63]:
```suggestion
7. Click **Validate**.
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+2. Click the **email** of the user you want to give database access to.
+
+3. Open the **Groups & Policies** tab.
+
+4. Click **+ Attach a policy**. The policy attachment wizard displays.
+
+5. Choose **Select an unassigned policy**, then, select the previously created policy from the drop-down menu.
+
+    <Message type="note">
+        You can also [create a new policy](serverless/sql-databases/how-to/manage-permissions/) or duplicate an existing one to manage the permissions of a user.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:41]:
```suggestion
        You can also [create a new policy](/serverless/sql-databases/how-to/manage-permissions/) or duplicate an existing one to manage the permissions of a user.
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+---
+
+Users can connect to a database and access its data. Each one can have a different set of customizable permissions.
+
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a Serverless SQL Database](/serverless/sql-databases/how-to/create-a-database/)
+  - You have [created a policy to manage users' permissions](serverless/sql-databases/how-to/manage-permissions/)
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:24]:
```suggestion
  - You have [created a policy to manage users' permissions](/serverless/sql-databases/how-to/manage-permissions/)
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+  - You have [invited a user to your organization](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/invite-user-to-orga/) or [created an application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-application/) in IAM
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:25]:
```suggestion
  - You have [invited a user to your organization](/identity-and-access-management/iam/how-to/invite-user-to-orga/) or [created an application](/identity-and-access-management/iam/how-to/create-application/) in IAM
```

File: serverless/sql-databases/how-to/add-users.mdx [hunk 9]
```diff
@@ -0,0 +1,64 @@
...
+3. Click the **name** of the application you want to add to the database.
+
+4. Click the **Groups & Policies** tab.
+
+5. Click **+ Attach a policy**. The policy attachment wizard displays.
+
+6. Choose **Select an unassigned policy**, then, select the previously created policy from the drop-down menu.
+
+    <Message type="note">
+        You can also [create a new policy](serverless/sql-databases/how-to/manage-permissions/) or duplicate an existing one to manage the permissions of an application.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/add-users.mdx:61]:
```suggestion
        You can also [create a new policy](/serverless/sql-databases/how-to/manage-permissions/) or duplicate an existing one to manage the permissions of an application.
```

File: serverless/sql-databases/how-to/configure-autoscaling.mdx [hunk 10]
```diff
@@ -0,0 +1,44 @@
...
+  tags: database serverless-sql-database serverless-db serverless-database postgresql 
+dates:
+  validation: 2023-07-21
+  posted: 2021-07-21
+categories:
+  - serverless
+  - sql-databases
+---
+
+This page shows you how to edit the autoscaling of a Serverless SQL Database using the Scaleway Console.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/configure-autoscaling.mdx:17]:
```suggestion
This page shows you how to edit the autoscaling of a Serverless SQL Database using the Scaleway console.
```

File: serverless/sql-databases/how-to/create-a-database.mdx [hunk 11]
```diff
@@ -0,0 +1,47 @@
...
+tags: database serverless-sql-database serverless-db serverless-database postgresql  
+dates:
+  validation: 2023-07-19
+  posted: 2021-07-19
+categories:
+  - serverless
+  - sql-databases
+---
+
+This page shows you how to create a Serverless SQL Database using the Scaleway Console.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/create-a-database.mdx:17]:
```suggestion
This page shows you how to create a Serverless SQL Database using the Scaleway console.
```

File: serverless/sql-databases/how-to/delete-a-database.mdx [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+tags: database 
+dates:
+  validation: 2023-07-21
+  posted: 2021-07-21
+categories:
+  - serverless
+  - sql-databases
+---
+
+This page shows you how to delete a Serverless SQL Database using the Scaleway Console.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/delete-a-database.mdx:17]:
```suggestion
This page shows you how to delete a Serverless SQL Database using the Scaleway console.
```

File: serverless/sql-databases/how-to/manage-permissions.mdx [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+    - **ServerlessSQLDatabaseFullAccess** allows the user to create and delete databases, edit their configuration, and to use the following commands:
+
+        - `SELECT`
+        - `UPDATE`
+        - `INSERT`
+        - `DELETE`
+        - `CREATE/ALTER/DROP TABLE`
+        - `CREATE/DROP INDEX`
+
+    - **ServerlessSQLDatabaseReadWrite** allows the user to use following commands:
```
Comment (jcirinosclwy) [inline serverless/sql-databases/how-to/manage-permissions.mdx:62]:
```suggestion
    - **ServerlessSQLDatabaseReadWrite** allows the user to use the following commands: 
```

File: serverless/sql-databases/how-to/connect-to-a-database.mdx [hunk 14]
```diff
@@ -0,0 +1,46 @@
...
+
+    - An [application](/identity-and-access-management/iam/concepts/#application) that you [previously created](/identity-and-access-management/iam/how-to/create-application/) in the IAM section of the console.
+
+    <Message type="note">
+        Refer to the [IAM documentation](/identity-and-access-management/iam/reference-content/overview/) for more information on identities and access rights.
+    </Message>
+
+3. Set up the **connection string** or **connection parameters**:
+
+    - If you already have a secret key, paste it in the connection text in place of the ``[INSERT_SECRET_KEY_HERE]`` placeholder.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/connect-to-a-database.mdx:42]: Not sure about this one though
```suggestion
    - If you already have a secret key, paste it in the connection text in place of the `[INSERT_SECRET_KEY_HERE]` placeholder.
```

File: serverless/sql-databases/how-to/connect-to-a-database.mdx [hunk 15]
```diff
@@ -0,0 +1,46 @@
...
+tags: database serverless-sql-database serverless-db serverless-database postgresql 
+dates:
+  validation: 2023-07-19
+  posted: 2021-07-19
+categories:
+  - serverless
+  - sql-databases
+---
+
+This page shows you how to set up the connection to a Serverless SQL Database using the Scaleway Console.
```
Comment (bene2k1) [inline serverless/sql-databases/how-to/connect-to-a-database.mdx:17]:
```suggestion
This page shows you how to set up the connection to a Serverless SQL Database using the Scaleway console.
```

File: serverless/sql-databases/how-to/manage-permissions.mdx [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+3. Click **Create a policy**. The policy creation wizard displays.
+
+4. Complete the steps of the first page of the creation wizard:
+
+    - Enter a **name** for the policy.
+
+    - Enter a **description** (optional).
+
+    - Select a **principal**, who will be the target of your policy. The [principal](/identity-and-access-management/iam/concepts/#principal) should be the user, application or group who you want to give access to your database.
```
Comment (jcirinosclwy) [inline serverless/sql-databases/how-to/manage-permissions.mdx:37]:
```suggestion
    - Select a **principal**, who will be the target of your policy. The [principal](/identity-and-access-management/iam/concepts/#principal) should be the user, application, or group who you want to give access to your database.
```

**Review by jcirinosclwy — APPROVED** [review]

File: serverless/sql-databases/how-to/configure-autoscaling.mdx [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+3. Click **Edit autoscaling**. A pop-up displays.
+
+4. Define the minimum and maximum vCPU you want to allocate to your database.
+
+        <Message type="note">
+            - Define a minimum of 1 vCPU or more to avoid cold starts.
+            - The minimum vCPU cannot be greater than the maximum vCPU.
+        </Message>
+
+5. Review the new [compute](link to concept) estimated cost.
```
Comment (RoRoJ) [inline serverless/sql-databases/how-to/configure-autoscaling.mdx:39]: missing link

File: serverless/sql-databases/how-to/manage-permissions.mdx [hunk 18]
```diff
@@ -0,0 +1,78 @@
...
+      - A policy can only be attached to one principal at a time.
+    </Message>
+
+5. Click **Add rules**. The second screen of the policy creation wizard.
+
+6. Select **Access to resources**, then, select the [Project](/identity-and-access-management/iam/concepts/#project) in which you want the permissions to apply.
+
+7. Click **Validate** to continue.
+
+8. Select **Serverless** in the left-hand **Products** list, then, tick the Permissions set you want to add to the policy:
```
Comment (RoRoJ) [inline serverless/sql-databases/how-to/manage-permissions.mdx:51]:
```suggestion
8. Select **Serverless** in the left-hand **Products** list, then, tick the permissions set you want to add to the policy:
```

File: serverless/sql-databases/how-to/connect-to-a-database.mdx [hunk 19]
```diff
@@ -0,0 +1,46 @@
...
+        Refer to the [IAM documentation](/identity-and-access-management/iam/reference-content/overview/) for more information on identities and access rights.
+    </Message>
+
+3. Set up the **connection string** or **connection parameters**:
+
+    - If you already have a secret key, paste it in the connection text in place of the ``[INSERT_SECRET_KEY_HERE]`` placeholder.
+
+    - If you do not have a secret key, click **Generate new secret key**.
+
+4. Copy your connection string or parameter and store it securely.
```
Comment (RoRoJ) [inline serverless/sql-databases/how-to/connect-to-a-database.mdx:46]: It seems like there's a part of this page missing which explains how to actually use your secret key to connect to the DB?

File: serverless/sql-databases/how-to/add-users.mdx [hunk 20]
```diff
@@ -0,0 +1,64 @@
...
+<Macro id="iam-requirements" />
+ 
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have [created a Serverless SQL Database](/serverless/sql-databases/how-to/create-a-database/)
+  - You have [created a policy to manage users' permissions](serverless/sql-databases/how-to/manage-permissions/)
+  - You have [invited a user to your organization](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/invite-user-to-orga/) or [created an application](https://www.scaleway.com/en/docs/identity-and-access-management/iam/how-to/create-application/) in IAM
+</Message>
+
+## How to add an existing user to a Serverless SQL Database
```
Comment (RoRoJ) [inline serverless/sql-databases/how-to/add-users.mdx:28]: Maybe explain what is meant by "existing user" - presumably someone who already has a SCW account? What if the person wants to add a non-existing user?

File: serverless/sql-databases/how-to/connect-to-a-database.mdx (hunk 19 above)
Comment (SamyOubouaziz) [reply serverless/sql-databases/how-to/connect-to-a-database.mdx:46]: It depends a lot on the environment, there are several ways to access a DB and perform operations in it, but it seems pretty obvious for developpers an engineers

Comment (SamyOubouaziz) [reply serverless/sql-databases/how-to/connect-to-a-database.mdx]: procedure for user added, generic message for applications has been added

**Pull request merged**